sql/plan/call.go

```diff
@@ -32,15 +33,22 @@ var _ sql.Node = (*Call)(nil)
 var _ sql.Expressioner = (*Call)(nil)
 
 // NewCall returns a *Call node.
-func NewCall(name string, params []sql.Expression) *Call {
+func NewCall(db sql.Database, name string, params []sql.Expression) *Call {
 	return &Call{
+		db:     db,
 		Name:   name,
 		Params: params,
 	}
 }
 
 // Resolved implements the sql.Node interface.
 func (c *Call) Resolved() bool {
+	if c.db != nil {
+		_, ok := c.db.(sql.UnresolvedDatabase)
+		if ok {
+			return false
+		}
```

Comment:
(minor) Calling the `Resolved()` method on `c.db` would be a little more consistent with the rest of the codebase.

```suggestion
                if !c.db.Resolved() {
			return false
		}
```

Reply:
`c.db` is of type `sql.Database`, and that type does not implement `Resolvable`.

It also looks like the consistent `Resolved` implementation is to do this kind of cast to check if the database is resolved, for instance in [create_role.go](https://github.com/dolthub/go-mysql-server/blob/main/sql/plan/create_role.go#L75-L79):
```
// Resolved implements the interface sql.Node.
func (n *CreateRole) Resolved() bool {
	_, ok := n.MySQLDb.(sql.UnresolvedDatabase)
	return !ok
}
```

Reply:
Ahh... thanks for double checking. This looks fine to me!